db/revtree.go

```diff
@@ -381,6 +377,20 @@ func ParseRevisions(body Body) []string {
 	return result
 }
 
+// Splits out the "start" and "ids" properties from encoded revision list
+func splitRevisionList(revisions Body) (int, []string) {
+	start, ok := base.ToInt64(revisions["start"])
+	digests, _ := GetStringArrayProperty(revisions, "ids")
```

Comment:
Do we need to worry about getting an error back from GetStringArrayProperty here, or should callers know they have a well-formed revisions Body by the time they call this.